package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cm-page-builder",
-  "version": "1.2.39",
+  "version": "1.2.40",
   "description": "Awesome react starter kit",
```

Comment:
@chandan-reddy-k @Surendhar95 - can you change the description of this please?

Reply:
Sure, do you have one ?